packages/hash/frontend/src/lib/user.ts

```diff
@@ -108,9 +109,19 @@ export const constructAuthenticatedUser = (params: {
       ({ value }) => value === primaryEmailAddress,
     )?.verified === true;
 
+  /**
+   * @todo: determine whether an authenticated user is an instance admin from the subgraph
+   * when querying the incoming links of an entity rooted subgraph is supported
+   *
+   * @see https://app.asana.com/0/1202805690238892/1203250435416416/f
+   */
+  const isInstanceAdmin = false;
```

Comment:
Is this fine for now because we are not using this fact for anything in the rest of the workspace API?

Reply:
If you think I should try and make that clearer in the above comment let me know

Reply:
Oh rereading your comment that should've been clear to me, apologies - I thought there was a misunderstanding why it was hardcoded to `false`.

So to answer your question it isn't required by anything in this PR, but I think it will be required once we start implementing the FE of the instance settings panel. Hopefully the blocking task will be completed before that though.